_includes/main_nav_tr.html

```diff
@@ -1,7 +1,14 @@
  <nav class="navbar navbar-expand-lg navbar-light bg-light fixed-top pl-md-5 pr-md-5 pl-sm-3 pr-0">
 
-    <a class="navbar-brand" href="{{ site.url }}/{{ page.lang }}/"><img src="{{ site.url }}/images/bisq-logo.svg" height="30"/></a>
+    <div>
+      <a class="navbar-brand" href="{{ site.url }}/{{ page.lang }}/"><img src="{{ site.url }}/images/bisq-logo.svg" height="30"/></a>
 
+      <!-- Outdated translation tag-->
+      {% if page.lang == page.lang == "de" or page.lang="es" or "pt-PT" or page.lang == "zh-CN" %}
```

Comment:
When is this "outdated translation" text supposed to show? This line seems to indicates it will show any time the current language is not EN (and indeed that's what happens when I run this locally).

Otherwise, please double-check the logic. `page.lang == page.lang` is redundant and you're missing the equality operator for ES (`page.lang="es"` needs `==`).

Reply:
My idea is that we'd regularly update it by taking out and leaving the relevant ones.